src/Distributions.jl

```diff
@@ -10,11 +10,6 @@ using Compat
 import Compat.view
 import Base.Random
 import Base: size, eltype, length, full, convert, show, getindex, scale, scale!, rand, rand!
-if VERSION < v"0.5.0-"
-    export shape
-else
-    import Base.shape
```

Comment:
maybe it did temporarily? ref https://github.com/JuliaStats/Distributions.jl/pull/506, cc @ranjanan when you do things that depend on `VERSION` please be as specific as you can - or better, use `isdefined`